js/privatebin.js

```diff
@@ -1131,6 +1134,21 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
         }
 
         /**
+         * display remaining message
+         *
+         * This automatically passes the text to I18n for translation.
+         *
+         * @name   Alert.showRemaining
+         * @function
+         * @param  {string|array} message     string, use an array for %s/%d options
+         */
+        me.showRemaining = function(message)
+        {
+            console.error('remaining message shown: ', message);
+            handleNotification(1, $remainingTime, message);
```

Comment:
Ah right, good idea. Seems I missed to use the notification handler for this message.